### ISSUE TITLE
Bluetrade create_limit_sell_order error

### ISSUE THREAD

**Issue by gaardiolor** [ccxt/ccxt#1226]
- OS: Windows
- Programming Language version: Python 3.6
- CCXT version: 671
- Exchange: Bleutrade
- Method: create_limit_sell_order

- Gererates an TypeError: TypeError: list indices must be integers or slices, not str
- Does stuff with bittrex.py ?
- The order seems to go through successfully despite this error (hence the 'insufficient funds' message. But I got the same error the first time)

```
import ccxt
from pprint import pprint
import test2_config as config

e = ccxt.bleutrade()
e.verbose = True
e.apiKey = config.exchange_keys['bleutrade']['key']
e.secret = config.exchange_keys['bleutrade']['secret']
pprint(e.create_limit_sell_order('ETH/BTC', float(0.001), 0.08920007))

C:\Users\gaardiolor\Anaconda3\python.exe C:/Users/gaardiolor/PycharmProjects/Test2/_test/test_bleutrade.py
GET https://bleutrade.com/api/v2/public/getmarkets 
Request: {'User-Agent': 'python-requests/2.18.4', 'Accept-Encoding': 'gzip, deflate'} None
GET https://bleutrade.com/api/v2/public/getmarkets 
Response: {'Date': 'Fri, 12 Jan 2018 21:24:10 GMT', 'Content-Type': 'text/html; charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '<snip>; expires=Sat, 12-Jan-19 21:24:09 GMT; path=/; domain=.bleutrade.com; HttpOnly', 'Vary': 'Accept-Encoding', 'Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Credentials': 'true', 'Server': 'cloudflare', 'CF-RAY': '<snip>', 'Content-Encoding': 'gzip'} {"success":"true","message":"","result":<sniplooksgood>
GET https://bleutrade.com/api/v2/public/getcurrencies 
Request: {'User-Agent': 'python-requests/2.18.4', 'Accept-Encoding': 'gzip, deflate'} None
GET https://bleutrade.com/api/v2/public/getcurrencies 
Response: {'Date': 'Fri, 12 Jan 2018 21:24:10 GMT', 'Content-Type': 'text/html; charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '<snip>; expires=Sat, 12-Jan-19 21:24:10 GMT; path=/; domain=.bleutrade.com; HttpOnly', 'Vary': 'Accept-Encoding', 'Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Credentials': 'true', 'Server': 'cloudflare', 'CF-RAY': '<snip>', 'Content-Encoding': 'gzip'} {"success":"true","message":"","result":<sniplooksgood>
GET https://bleutrade.com/api/v2/market/selllimit?nonce=1515792250&apikey=<snip>&market=ETH_BTC&quantity=0.001&rate=0.08920007 
Request: {'apisign': '<snip>', 'User-Agent': 'python-requests/2.18.4', 'Accept-Encoding': 'gzip, deflate'} None
GET https://bleutrade.com/api/v2/market/selllimit?nonce=1515792250&apikey=<snip>&market=ETH_BTC&quantity=0.001&rate=0.08920007 
Response: {'Date': 'Fri, 12 Jan 2018 21:24:12 GMT', 'Content-Type': 'text/html; charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '<snip>; expires=Sat, 12-Jan-19 21:24:10 GMT; path=/; domain=.bleutrade.com; HttpOnly', 'Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Credentials': 'true', 'Server': 'cloudflare', 'CF-RAY': '<snip>', 'Content-Encoding': 'gzip'} {"success":"false","message":"Insufficient funds!","result":[]}
Traceback (most recent call last):
  File "C:/Users/gaardiolor/PycharmProjects/Test2/_test/test_bleutrade.py", line 9, in <module>
    pprint(e.create_limit_sell_order('ETH/BTC', float(0.001), 0.08920007))
  File "C:\Users\gaardiolor\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 1015, in create_limit_sell_order
    return self.create_order(symbol, 'limit', 'sell', *args)
  File "C:\Users\gaardiolor\Anaconda3\lib\site-packages\ccxt\bittrex.py", line 419, in create_order
    'id': response['result']['uuid'],
TypeError: list indices must be integers or slices, not str

Process finished with exit code 1
```

**Comment by gaardiolor** [comment]
create_market_sell_order also doesn't work, maybe related? Also references of bittrex.py. 

```
import ccxt
from pprint import pprint
import test2_config as config

e = ccxt.bleutrade()
e.verbose = True
e.apiKey = config.exchange_keys['bleutrade']['key']
e.secret = config.exchange_keys['bleutrade']['secret']
pprint(e.create_market_sell_order('ETH/BTC', float(0.001), 0.08920007))


C:\Users\gaardiolor\Anaconda3\python.exe C:/Users/gaardiolor/PycharmProjects/test2/_test/test_bleutrade.py
GET https://bleutrade.com/api/v2/public/getmarkets 
Request: {'User-Agent': 'python-requests/2.18.4', 'Accept-Encoding': 'gzip, deflate'} None
GET https://bleutrade.com/api/v2/public/getmarkets 
Response: {'Date': 'Fri, 12 Jan 2018 21:23:13 GMT', 'Content-Type': 'text/html; charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '<snip>; expires=Sat, 12-Jan-19 21:23:12 GMT; path=/; domain=.bleutrade.com; HttpOnly', 'Vary': 'Accept-Encoding', 'Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Credentials': 'true', 'Server': 'cloudflare', 'CF-RAY': '<snip>', 'Content-Encoding': 'gzip'} {"success":"true","message":"","result":<sniplooksgood>
GET https://bleutrade.com/api/v2/public/getcurrencies 
Request: {'User-Agent': 'python-requests/2.18.4', 'Accept-Encoding': 'gzip, deflate'} None
GET https://bleutrade.com/api/v2/public/getcurrencies 
Response: {'Date': 'Fri, 12 Jan 2018 21:23:15 GMT', 'Content-Type': 'text/html; charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '<snip>; expires=Sat, 12-Jan-19 21:23:13 GMT; path=/; domain=.bleutrade.com; HttpOnly', 'Vary': 'Accept-Encoding', 'Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Credentials': 'true', 'Server': 'cloudflare', 'CF-RAY': '<snip>', 'Content-Encoding': 'gzip'} {"success":"true","message":"","result":<sniplooksgood>
Traceback (most recent call last):
  File "C:/Users/gaardiolor/PycharmProjects/test2/_test/test_bleutrade.py", line 9, in <module>
    pprint(e.create_market_sell_order('ETH/BTC', float(0.001), 0.08920007))
  File "C:\Users\gaardiolor\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 1021, in create_market_sell_order
    return self.create_order(symbol, 'market', 'sell', amount, None, params)
  File "C:\Users\gaardiolor\Anaconda3\lib\site-packages\ccxt\bittrex.py", line 416, in create_order
    response = getattr(self, method)(self.extend(order, params))
  File "C:\Users\gaardiolor\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 473, in extend
    result.update(arg)
TypeError: 'float' object is not iterable

Process finished with exit code 1
```

**Comment by gaardiolor** [comment]
So I've copied create_order from bittrex.py and changed 
`'id': response['result']['uuid'],`
to 
`'id': response['result']['orderid'],`

that seems to fix it

**Comment by kroitor** [comment]
Fixed in 1.10.674. Thank you for reporting!
Be aware: both Bittrex and Bleutrade support **limit orders only**.
